_docs/_latest/configure/explain/index.md

```diff
@@ -1,135 +1,59 @@
 ---
 title: Explain
-order: 202
+order: 000
 ---
 # {{ page.title }}
 
-Explain is not part of the main Forseti releases from master. Please
-deploy from the Explain experimental branch.
+This guide describes how to set up Explain for Forseti Security.
+Explain helps you understand the Cloud Identity and Access Management
+(Cloud IAM) policies that affect your Google Cloud Platform (GCP) resources.
+It can enumerate access by resource or member, answer why a principal has access 
+to a certain resource, or offer possible strategies for how to grant a specific 
+resource.
 
-This page describes how to set up Explain for Forseti Security.
-Explain is a client-server based application that helps administrators,
-auditors, and users of Google Cloud to understand, test, and develop Cloud
-Identity and Access Management (Cloud IAM) policies. It can enumerate access by
-resource or member, answer why a principal has access to a certain resource, or
-offer possible strategies for how to grant a specific resource. You'll use a
-command-line interface to deploy Explain on a dedicated virtual machine.
+## Running Explain
 
-For more detail about your Cloud IAM policies, you can use the denormalizer,
-which calculates all the principals, permissions, and resources for the model.
-It also allows a primitive "diff" of the access between two full models, such
-as comparing the access at two different points in time. To use the
-denormalizer, run `forseti_iam explainer denormalize` or
-`forseti_iam explainer --help`. You can also explore the `forseti_iam` tool by
-using `forseti_iam --help`.
+Before you start using Explain, you'll first select the data model you
+want to use.
 
-Because Explain and its API are still in early development, third party
-clients won't be supported until a first stable API version is released.
+To review the hierarchy of commands and explore Explain functionality, use
+`–help`.
 
-## Deploying Explain 
+### Selecting a data model
```

Comment:
I think we need a step on creating a data model and it's pre-requisites (a working version of Forseti with at least one inventory.)

Reply:
Updated.